master/internal/rm/agentrm/agent_resource_manager.go

```diff
@@ -735,3 +754,19 @@ func (a ResourceManager) NotifyContainerRunning(
 	return errors.New(
 		"the NotifyContainerRunning message is unsupported for AgentResourceManager")
 }
+
+type taskContainerDefaults struct {
+	fallbackDefault model.TaskContainerDefaultsConfig
+	resourcePool    string
+}
+
+// TaskContainerDefaults returns TaskContainerDefaults for the specified pool.
+func (a ResourceManager) TaskContainerDefaults(
+	ctx actor.Messenger,
+	pool string,
+	defaultConfig model.TaskContainerDefaultsConfig,
+) model.TaskContainerDefaultsConfig {
+	request := taskContainerDefaults{fallbackDefault: defaultConfig, resourcePool: pool}
+	response := a.Ref().System().Ask(a.Ref(), request).Get()
+	return response.(model.TaskContainerDefaultsConfig)
```

Comment:
oh oops, this should use a.Ask

Reply:
same in the dispatcher ee code. if you peruse the code for https://github.com/determined-ai/determined/blob/master/master/internal/rm/actorrm/actor_resource_manager.go#L265-L291 it should sort of explain why (what it handles that this doesn't)